api/audit-trail-gw/src/database/channel-info.ts

```diff
@@ -27,8 +27,8 @@ export const searchChannelInfo = async (channelInfoSearch: ChannelInfoSearch): P
 
 	const nameFilter = name ? { name: regex(name) } : undefined;
 	const authorFilter = authorId ? { authorId: regex(authorId) } : undefined;
-	const typeFilter = topicType ? { 'topics.type': { $regex: topicType } } : undefined;
-	const sourceFilter = topicSource ? { 'topics.source': { $regex: topicSource } } : undefined;
+	const typeFilter = topicType ? { 'topics.type': { $regex: regex(topicType) } } : undefined;
```

Comment:
just for clarification why do we need to write 
`{'topics.type': { $regex: regex(topicType) }}`. instead of `{'topics.type': regex(topicType) }` as done for instance for the name `{ name: regex(name) }`

Reply:
I would prefer to make it consistent either way.